GOVERNANCE.md

```diff
@@ -0,0 +1,57 @@
+# `userland-migrations` Team Governance
+
+Meet the current team members of the Node.js userland migrations:
+
+- [@AugustinMauroy](https://github.com/AugustinMauroy) - **Augustin Mauroy** (he/him)
+- [@avivkeller](https://github.com/avivkeller) - **Aviv Keller** (he/him)
+- [@bmuenzenmeyer](https://github.com/bmuenzenmeyer) - **Brian Muenzenmeyer** (he/him)
+- [@JakobJingleheimer](https://github.com/JakobJingleheimer) - **Jacob Smith** (he/him) - Maintainer
+- [@ljharb](https://github.com/ljharb) - **Jordan Harband** (he/him)
+
+## Rights
+
+Any team member can block a pull request or decision. A block can be overturned by a team vote, which requires a quorum of two-thirds of active members and passes with a simple majority (>50%). Pursuant to Node.js policy, the [TSC](https://github.com/nodejs/TSC) may override a team’s vote.
```

Comment:
I'd specify that since this is not working group, any collaborator can also block a pr or decision.